eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -250,10 +251,72 @@ pparamUpdateSpec =
 
 proposalsWithVotingSpec ::
   forall era.
-  ConwayEraImp era =>
+  ( ConwayEraImp era
+  , InjectRuleFailure "LEDGER" ConwayGovPredFailure era
+  ) =>
   SpecWith (ImpTestState era)
 proposalsWithVotingSpec =
   describe "Proposals" $ do
+    describe "Failures expected after bootstrap phase" $ do
+      it "Fails predicate when treasury withdrawal has nonexistent return address" $ do
+        policy <- getGovPolicy
+        protVer <- getProtVer
+        registeredRewardAccount <- registerRewardAccount
+        netId <- use (impGlobalsL . to networkId)
+        unregisteredRewardAccount <- RewardAccount netId . KeyHashObj <$> freshKeyHash
```

Comment:
You could do this if you like, to avoid dealing with impGlobals and network ids: 
```suggestion
        unregisteredRewardAccount <- freshKeyHash >>= getRewardAccountFor . KeyHashObj
```